NEOSPlus/Components/Wizards/ComponentBreakdown.cs

```diff
@@ -0,0 +1,121 @@
+﻿// MIT License
```

Comment:
I'm not super familiar with licenses, @Xlinka should look at this and say if this is correct

Reply:
The wording or integration into this project?

Reply:
Integration, I'm not sure how having MIT portions within LGPL works

Reply:
this is actually how it should be done.